src/js/controllers/NewDisasterNotificationCtrl.js

```diff
@@ -0,0 +1,53 @@
+function NewDisasterNotificationCtrl($scope, $location, $route, $routeParams, $http, profileService, operations, profileData) {
+  if($routeParams.profileId && !profileData.profile){
+    // No profile data
+    return false;
+  }
+  $scope.operations = operations;
+  var locationPath = $location.$$path.split("/");
+  if (profileData.contact && profileData.profile) {
+    profileData.contact._profile = profileData.profile;
+  }
+  $scope.automaticAddDisaster = false;
+  if(locationPath.length == 4)
+    $scope.automaticAddDisaster = true;
+
+  if($scope.automaticAddDisaster){
```

Comment:
This check is useless... When you get there, automaticAddDisaster is always true